cmd/create.go

```diff
@@ -139,6 +145,7 @@ A full list of component types that can be deployed is available using: 'odo cat
 			checkError(err, "")
 			fmt.Printf("Please wait, creating %s component ...\n", componentName)
 			err = component.Build(client, componentName, applicationName, false, true, stdout)
+			fmt.Println("build failed", err)
```

Comment:
shouldn't this check if there is an error or not first?
Also, there is a `checkError` function ;-)

Reply:
:see_no_evil: